runtime/lua/vim/lsp/util.lua

```diff
@@ -1160,7 +1162,7 @@ end
 --- Shows document and optionally jumps to the location.
 ---
 ---@param location table (`Location`|`LocationLink`)
----@param offset_encoding "utf-8" | "utf-16" | "utf-32"
+---@param offset_encoding string utf-8|utf-16|utf-32
```

Comment:
Can we leave this with the union type/enum syntax instead of generalizing it to string? Only the 3 values are allowed, not a string in general.